go/libraries/doltcore/remotestorage/noop_chunk_cache.go

```diff
@@ -29,21 +29,21 @@ var noopChunkCache = &noopChunkCacheImpl{}
 type noopChunkCacheImpl struct {
 }
 
-func (*noopChunkCacheImpl) Put(chnks []chunks.Chunkable) {
+func (*noopChunkCacheImpl) Put(chnks []nbs.CompressedChunk) {
 }
 
-func (*noopChunkCacheImpl) Get(hashes hash.HashSet) map[hash.Hash]chunks.Chunkable {
-	return make(map[hash.Hash]chunks.Chunkable)
+func (*noopChunkCacheImpl) Get(hashes hash.HashSet) map[hash.Hash]nbs.CompressedChunk {
+	return make(map[hash.Hash]nbs.CompressedChunk)
```

Comment:
This was broken when it returned `map[hash.Hash]chunks.Chunkable` because `chunks.Chunkable` zero value is `nil`, and caller is expecting non-`nil` zero value where `v.IsEmpty() == true`.

Reply:
Maybe make this a member so you don't have to make a new one for every get call?

Reply:
Decided to leave it.